README.md

```diff
@@ -48,12 +48,7 @@ env:
 > will trigger a GitHub Workflow that will block merging of your Pull Request until changes are effected.
 
 ## Versioning
-When updating a dataset in `xclim-testdata` using a development branch and Pull Request, once changes have been merged
-to the `main` branch, you should tag a new version of `xclim-testdata`. The version tag of `xclim-testdata` is loosely tied
-to that of `xclim`, and should represent the `xclim` release (**stable**, i.e. version string not ending in `-dev`)
-that is compatible with the latest changes in `xclim-testdata`.
-
-> **Warning** 
-> In the event that multiple Pull Requests occur for `xclim-testdata` between two stable versions of `xclim`
-> (e.g. one for `v1.2.3-dev` and another for `v1.2.5-dev`), the `xclim-tesdata` tag (e.g. `v1.3.0`)
-> should be replaced with the commit reflecting the most recent changes.
+When updating a dataset in `xclim-testdata` using a development branch and Pull Request,
+once changes have been merged to the `main` branch, you should tag a new version of `xclim-testdata`. 
+
+The version tag of `xclim-testdata` should follow a [calendar versioning](https://calver.org/) scheme (i.e. version string follows from `YYYY.MM.DD`, without modifiers) reflecting the date of the tag creation.
```

Comment:
"version string follows from `YYYY.MM.DD`, without modifiers", I wonder, if you release twice in the same day because you needed a hot fix, how will you do this?  So the part about "without modifiers" seems too strict to me.

Maybe allow `YYYY.MM.DD-r#` if ever we need a quick hot fix.

Reply:
Reasonable. The code in the xclim branch does support modifiers. Will adjust.